### PR TITLE
Unit test cases

### PR DESCRIPTION
**Pull request by rishubhjain**
tendrl-bug-id: Tendrl/ceph-integration#62

### CODE REVIEW

File: tendrl/ceph_integration/tests/fixtures/client.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Client():
```
Comment (shtripat) [inline tendrl/ceph_integration/tests/fixtures/client.py]: This style of class gives a pep8 error. Extend from `object`. Applies to all the instances.

File: tendrl/ceph_integration/tests/fixtures/client.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    def __init__(self):
+        pass
+    def read(*args,**kwargs):
```
Comment (shtripat) [inline tendrl/ceph_integration/tests/fixtures/client.py]: Better run `tox -epep8` and resolve the issues.

File: tendrl/ceph_integration/tests/test_types.py [hunk 3]
```diff
@@ -86,12 +89,24 @@ def setup_method(self, method):
         self.osd_map_data.__getitem__.side_effect = lambda x: \
             self.data[x] if x in self.data else self.osd_map_data
 
+    @mock.patch('tendrl.commons.event.Event.__init__',
```
Comment (shtripat) [inline tendrl/ceph_integration/tests/test_types.py:92]: I find below code more readable 

```
@mock.patch(
    'tendrl.commons.event.Event.__init__',
    mock.Mock(return_value=None)
)
```

If it works for you :)

**Comment by r0h4n** [comment]
@rishubhjain Fix pep8 issues for this PR, need to merge this one

**Pull request closed without merging**